great_expectations/expectations/expectation.py

```diff
@@ -1863,6 +1863,17 @@ def validate_metric_value_between_configuration(
         except AssertionError as e:
             raise InvalidExpectationConfigurationError(str(e))
 
+        # Only test for min <= max if both values are numeric. If either is a dictionary,
```

Comment:
@jcampbell Should there be the check for the max_val < min_val end of the range as well?  Thanks.

Reply:
Sorry -- this shouldn't even be in this PR, I think that was a mistake; must not have cleaned my working directory from a different experiment.